test/run-tests

```diff
@@ -677,7 +708,7 @@ def make_html(source_file):
     return html_file
 
 
-def handle_task(gh, args, config, task, ansible_id, dry_run=False):
+def handle_task(gh, args, config, task, ansible_id, dry_run=False, openstack=False):
```

Comment:
you don't need a separate parameter here since there is already `args.openstack`